app/views/local_authorities/index.html.erb

```diff
@@ -8,30 +8,41 @@
 
 <%= render partial: 'shared/home_nav' %>
 
-<table class="table table-bordered table-with-counts" data-module="filterable-table">
-  <thead>
-    <tr class="table-header">
-      <th>
-        <div class='filter-control'>
-          <%= render partial: 'govuk_admin_template/table_filter_form', locals: { placeholder: 'Search for a council here' } %>
-        </div>
-        <div class='count'>
-          <span><%= @authorities.count %></span>
-          <%= 'local authorities'.pluralize(@authorities.count) %>
-        </div>
-      </th>
-    </tr>
-  </thead>
-  <tbody>
-    <% @authorities.each do |authority| %>
-      <tr>
-        <td>
-          <%= link_to authority.name, local_authority_path(authority.slug, filter: 'broken_links'), class: 'js-open-on-submit name' %>
-          <div class="center count <%= singular_or_plural(authority.broken_link_count) %>">
-            <span><%= authority.broken_link_count %></span>
+<div data-module="track-search"
+     data-track-action="filterUsed"
+     data-track-label="filterUsed"
+     data-track-category="userInteraction:LLM">
+  <table class="table table-bordered table-with-counts" data-module="filterable-table">
+    <thead>
+      <tr class="table-header">
+        <th>
+          <div class='filter-control'>
+            <%= render partial: 'govuk_admin_template/table_filter_form', locals: { placeholder: 'Search for a council here' } %>
           </div>
-        </td>
+
+          <div class="count">
+            <% content_for :custom_dimension_js do %>
```

Comment:
Why is this template variable being set here? I'd expect it to be set somewhere further up, at the start of a template or elsewhere.